demo/src/app/components/datepicker/demos/calendars/islamic-calendars.html

```diff
@@ -0,0 +1,26 @@
+<p><b>Islamic <i>{{name}}</i> Calendar</b></p>
+
+<div [(ngModel)]="language" ngbRadioGroup>
```

Comment:
Unfortunately the radio group markup changed a bit since the time PR was opened... please check https://ng-bootstrap.github.io/#/components/buttons/examples, you have to add `ngbButtonLabel` and `ngbButton` attributes

Moreover I'm not sure the language change will work this way, because we have added `ChangeDetection.OnPush` on the datepicker. We had to simplify i18n demo of the datepicker for this reason, removing the radio button